plugin/mark.vim

```diff
@@ -157,52 +157,54 @@ nnoremap <silent> <Plug>MarkSearchCascadeNextNoStop     :<C-u>if ! mark#cascade#
 nnoremap <silent> <Plug>MarkSearchCascadePrevNoStop     :<C-u>if ! mark#cascade#Next(v:count1, 0, 1)<Bar>execute "normal! \<lt>C-\>\<lt>C-n>\<lt>Esc>"<Bar>echoerr ingo#err#Get()<Bar>endif<CR>
 
 
-if !hasmapto('<Plug>MarkSet', 'n')
-	nmap <unique> <Leader>m <Plug>MarkSet
-endif
-if !hasmapto('<Plug>MarkSet', 'x')
-	xmap <unique> <Leader>m <Plug>MarkSet
-endif
-" No default mapping for <Plug>MarkIWhiteSet.
-if !hasmapto('<Plug>MarkRegex', 'n')
-	nmap <unique> <Leader>r <Plug>MarkRegex
-endif
-if !hasmapto('<Plug>MarkRegex', 'x')
-	xmap <unique> <Leader>r <Plug>MarkRegex
-endif
-if !hasmapto('<Plug>MarkClear', 'n')
-	nmap <unique> <Leader>n <Plug>MarkClear
-endif
-" No default mapping for <Plug>MarkAllClear.
-" No default mapping for <Plug>MarkConfirmAllClear.
-" No default mapping for <Plug>MarkToggle.
+if !exists("g:mark_no_mappings") || !g:mark_no_mappings
```

Comment:
This repo seems to use the CamelCase syntax. I would suggest to change the name of this variable accordingly.

Reply:
Hm. `g:no_x_maps` is a pretty common vim idiom (try typing `:let g:no_<Tab>` and see how many wild results you get), but you're right that it breaks consistency. I think the current use of `g:loaded_mark` as following convention justifies this break from consistency?